ert/data/record/_transformation.py

```diff
@@ -46,6 +49,14 @@ async def transform_input(
     async def transform_output(self, mime: str, location: Path) -> Record:
         return await _load_record_from_file(location, mime)
 
+    async def transform_output_sequence(
```

Comment:
No sure here, but can this function be a classmethod perhaps?

Reply:
In fact all methods can be static methods...

Reply:
This will change as for instance RecordTreeTransformation would need to get parameters (in the builder phase). So here I meant only this sequence function as it's a bit out of place.

Reply:
I see. Let's leave it for now, to keep the interface uniform.